samples/04_gis_analysts_data_scientists/determining_site_suitability_for_oil_palm_plantation.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>A '?' before or after an RFT object will display the help document consisting of the parameters marked as public by the author of the RFT.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>